app/assets/stylesheets/osem-schedule.css.scss

```diff
@@ -214,6 +214,15 @@ td.no-padding{
   display: inline-block;
 }
 
+.title{
+  font-size: 23px;
+}
+
+.subtitle{
```

Comment:
If you read a little bit further the docu tells you there are also heading classes. So you can do `p.h3` :-)